CITING.md

```diff
@@ -43,22 +36,57 @@ of Open Source Software, which can be cited with the following reference:
   author = {Hartwig Anzt and Terry Cojean and Yen-Chen Chen and Goran Flegar and Fritz G\"{o}bel and Thomas Gr\"{u}tzmacher and Pratik Nayak and Tobias Ribizel and Yu-Hsiang Tsai},
   title = {Ginkgo: A high performance numerical linear algebra library},
   journal = {Journal of Open Source Software}
-}
+
 ```
 
+For topical publications, please cite whichever features you use within Ginkgo:
+
+**[Portability](#on-portability)** |
+**[Software Sustainability](#on-software-sustainability)** |
+**[SpMV performance](#spmv-performance)** |
+**[Preconditioners](#preconditioners)** |
+**[Mixed-precision](#mixed-precision-algorithms)** |
+**[Sparse direct](#sparse-factorizations-and-direct-solvers)** |
+**[Batched](#batched-algorithms)**
+
+
 ### On Portability
 
 ``` bibtex
-@misc{tsai2020amdportability,
-    title={Preparing Ginkgo for AMD GPUs -- A Testimonial on Porting CUDA Code to HIP},
-    author={Yuhsiang M. Tsai and Terry Cojean and Tobias Ribizel and Hartwig Anzt},
-    year={2020},
-    eprint={2006.14290},
-    archivePrefix={arXiv},
-    primaryClass={cs.MS}
+@inproceedings{tsaiPreparingGinkgoAMD2021,
```

Comment:
I think the main paper which summarizes these efforts would be:

```@article{COJEAN2022102902,
title = {{Ginkgo—A math library designed for platform portability}},
journal = {{Parallel Computing}},
volume = {111},
pages = {102902},
year = {2022},
issn = {0167-8191},
doi = {https://doi.org/10.1016/j.parco.2022.102902},
url = {https://www.sciencedirect.com/science/article/pii/S0167819122000096},
author = {Terry Cojean and Yu-Hsiang Mike Tsai and Hartwig Anzt},
keywords = {Porting to GPU accelerators, Platform Portability, Performance portability, AMD, NVIDIA, Intel}
}
```


I generally think it's better to cite journals first or important conferences (e.g., IPDPS).